frontend/components/PageHeader.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC, ReactNode } from "react";
+
+interface PageHeaderProps {
+  children: ReactNode;
+}
+
+const PageHeader: FC<PageHeaderProps> = ({ children }: PageHeaderProps) => {
+  return (
+    <>
+      <h1 className="page-header text-center my-3">{children}</h1>
+      <hr />
+    </>
+  );
+};
+
+export default PageHeader;
```

Comment:
Could we replace the header on the SERP with this component? They look a bit different.
https://github.com/ModularHistory/modularhistory/blob/6ee5a052b250c0e24893fd635196d8acde90d026/frontend/pages/search.jsx#L74-L84
![image](https://user-images.githubusercontent.com/24942155/116837195-fdff3a00-ab8e-11eb-9b29-4871011b1e06.png)

Reply:
I do think it would be good to update the SERP header. This new component isn't a good drop-in replacement for what's being use now, though, so we should probably make a separate task for it.